tests/common/TestsSetup.hs

```diff
@@ -0,0 +1,42 @@
+-----------------------------------------------------------------------------
+--
+-- Module      :  Main
+-- Copyright   :  (c) Oleg Grenrus 2015
```

Comment:
For code which is licensed with a different copyright ascription, we've started requiring the license to be explicitly listed or linked here, to make it clear. Could you add a link to the MIT license if that's the one you're using?

Thanks.